templates/quickstart-confluence-master.template.yaml

```diff
@@ -524,7 +524,7 @@ Parameters:
   KeyPairName:
     ConstraintDescription: Must be the name of an existing EC2 Key Pair.
     Description: The EC2 Key Pair to allow SSH access to the instances. If you don't provide one, the Atlassian Standard Infrastructure stack's key pair will be used.
-    Type: String
+    Type: List<AWS::EC2::Keypair>
```

Comment:
This is the same case as https://github.com/atlassian/quickstart-atlassian-bitbucket/pull/36#discussion_r395424345